### ISSUE TITLE
[PATCH] ifimsg_vxlan.c:33:12: error: implicit declaration of function ‘parse_ifla_vxlan_id’; did you mean ‘parse_ifla_vlan_id’? [-Werror=implicit-function-declaration]

### ISSUE THREAD

**Issue by nabijaczleweli** [t2mune/nield#4]
Bug-Debian: https://bugs.debian.org/1066535

Patch:
```diff
@@ -317,6 +317,7 @@ const char *conv_macvlan_mode(unsigned mode, unsigned char debug);
 /* ifimsg_vxlan.c */
 #if HAVE_DECL_IFLA_VXLAN_UNSPEC
 int parse_ifla_vxlan(char *msg, char **mp, struct rtattr *info, struct iflist_entry *ifle);
+int parse_ifla_vxlan_id(char *msg, char **mp, struct rtattr *vxlan, struct iflist_entry *ifle);
 int parse_ifla_vxlan_link(char *msg, char **mp, struct rtattr *vxlan, struct iflist_entry *ifle);
 int parse_ifla_vxlan_local(char *msg, char **mp, struct rtattr *vxlan, struct iflist_entry *ifle, unsigned short family);
 int parse_ifla_vxlan_port_range(char *msg, char **mp, struct rtattr *vxlan, struct iflist_entry *ifle);
```

**Comment by t2mune** [comment]
This is a bug that has already been fixed. Could you please try again with v0.6.2?

**Comment by nabijaczleweli** [comment]
Yeah, apparently fixed in 3dbc9c49d497548ce7bcdae6d39ba1fab7679394 (but only released in v0.6.2).
